jekyll/install_unix.md

```diff
@@ -7,32 +7,126 @@ current: Install
 
 <h1 class="text-centered page-title main-heading">Install Nim on Unix</h1>
 
+There are several ways to install Nim on Unix-like systems, depending on
+your needs:
+
+* If you just want to use the stable version of Nim, you may find it simplest
+  to use your system's package manager.
```

Comment:
Maybe it is the simplest, but it is the surest way to install an outdated version of Nim.
E.g. I'm on Manjaro Linux, which is Arch derivative, which should have fairly recent stuff, and the version offered there is v0.18.0 (in few days it will be one year since it was released).

I would definitely add a note about the possibility of outdated version, and I'd like to remove it from the first place in the list. (The current version has "Installation using package managers" section in the last place)

Reply:
Good points! Done.